src/wallet/ClaimRewardsBlock.js

```diff
@@ -55,7 +55,7 @@ class ClaimRewardsBlock extends Component {
             rewardClaimed: true,
           });
           this.props.getUserAccountHistory(name);
-          this.props.updateAuthUser(name);
+          setTimeout(() => this.props.updateAuthUser(name), 2000);
```

Comment:
Shouldn't we just chain this after `getUserAccountHistory` succeeded instead of timeouts?
`this.props.getUserAccountHistory(name).then(...);`

Reply:
good idea will update